Gemfile

```
@@ -60,6 +60,8 @@ group :development, :test do
 
   gem 'brakeman', '~> 5.2.3', require: false
   gem 'bullet', '~> 7.0.2'
+  gem 'faker'
+  gem 'factory_bot_rails'
```

```suggestion
  gem 'faker', '~> 2.21.0'
  gem 'factory_bot_rails', '~> 6.2.1'
```

In the long run, using pessimistic versioning (meaning that we specify all [SEMVER](https://semver.org/) slots, will help us in upgrading the system. Whenever we run `bundle update xyz` it will update all dependencies according to the rules we set, which avoids unexpected surprises (like bugs caused by upgrading major versions without being aware).

I do know, though, that it sounds very pedantic, so I'll leave the decision to use it or not to you :)

---

I don't think it sounds pedantic at all! Seems like a good practice to me :]